apps/examples/src/2-api/APIExample.tsx

```diff
@@ -13,7 +13,7 @@ import { useEffect } from 'react'
 export default function APIExample() {
 	const handleMount = (editor: Editor) => {
 		// Create a shape id
-		const id = editor.createShapeId('hello')
```

Comment:
`editor.createShapeId` is here replaced by `createShapeId`, which it was previously calling internally